docs/make.jl

```diff
@@ -4,6 +4,7 @@ using EDF
 makedocs(; modules=[EDF],
          sitename="EDF.jl",
          authors="Beacon Biosignals, Inc.",
+         checkdocs=:public,
          pages=["API" => "index.md"])
 
-deploydocs(; repo="github.com/beacon-biosignals/EDF.jl.git")
+deploydocs(; repo="github.com/beacon-biosignals/EDF.jl.git", push_preview=true)
```

Comment:
Do the previews get garbage collected automatically? I feel like I remember the Julia website encountering a situation where a ton of PR preview branches were left around.

Reply:
There is no extra branch, it's just extra data on the `gh-pages` branch. For this repo, I don't think that's a problem -- there are no images or anything and so everything will diff/compress very nicely. On repos where you generate preview images, it's good to set up a GC action, which I'll add here, but I don't think it'll make a big difference.

Reply:
9449b8d